ui/copy-fusor-ember-cli-to-assets

```diff
@@ -0,0 +1,20 @@
+ #!/bin/bash
+
+cp ../fusor-ember-cli/dist/assets/vendor.css          ./app/assets/stylesheets/fusor_ui/vendor-fusor.css;
+cp ../fusor-ember-cli/dist/assets/fusor-ember-cli.css ./app/assets/stylesheets/fusor_ui/;
+
+cp ../fusor-ember-cli/dist/assets/vendor.js       ./app/assets/javascripts/fusor_ui/vendor-fusor.js;
+cp ../fusor-ember-cli/dist/assets/fusor-ember-cli.js   ./app/assets/javascripts/fusor_ui/;
+
+cp ../fusor-ember-cli/dist/assets/rhci-rhev-640-210.png   ./app/assets/images/;
+cp ../fusor-ember-cli/dist/assets/rhci-openstack-640-210.png   ./app/assets/images/;
+cp ../fusor-ember-cli/dist/assets/rhci-cloudforms-640-210.png  ./app/assets/images/;
+
+cp ../fusor-ember-cli/dist/assets/ovirt-640-210.png   ./app/assets/images/;
+cp ../fusor-ember-cli/dist/assets/rdo-640-210.png   ./app/assets/images/;
+cp ../fusor-ember-cli/dist/assets/manageiq-640-210.png  ./app/assets/images/;
+
+cp ../fusor-ember-cli/dist/fonts/*.*  ./public/fonts/;
+
+git add .;
+git commit -m "moved assets from /fusor-ember-cli/dist to /ui/app/assets";
```

Comment:
Yeah I understand, still having two servers seems unnecessary and unusual. Googling revealed a solution https://speakerdeck.com/rwjblue/rails-plus-ember-dot-js-plus-ember-cli-equals?slide=27, does not seem too complicated.

Let's do a separate PR.